tests/gui/conftest.py

```diff
@@ -0,0 +1,45 @@
+"""
+pytest configuration for GridSync's GUI tests.
+"""
+
+from unittest.mock import MagicMock
+
+import pytest
+from PyQt5.QtWidgets import QApplication
+from twisted.python.filepath import FilePath
+
+from gridsync.gui import Gui
+from gridsync.preferences import Preferences
+
+# Without this, QWidget() aborts the process with "QWidget: Must construct a
+# QApplication before a QWidget".
+app = QApplication([])
```

Comment:
FWIW, this is only necessary if you're running the `gui` tests in isolation (e.g., via `pytest tests/gui` or similar). If the test suite is run in its entirety (e.g., by running just `pytest` or `pytest tests` in the top level of the source tree), the `pytest-qt` plugin will initialize and use it's own `QApplication` instance throughout the run. I'm not sure why it doesn't work the same way in both instances, however... Anyway, there's no harm in keeping this one here; (Py)Qt will just use whichever `QApplication` instance is available first.